src/App.js

```diff
@@ -1,9 +1,9 @@
 import React, { Component } from 'react';
 
 
-import Header from './components/Header';
-import Navbar from './components/Navbar';
-import Profile from './components/Profile';
+import Header from './components/Header/Header';
+import Navbar from './components/Navbar/Navbar';
+import Profile from './components/Profile/Profile';
 
```

Comment:
Consider removing redundant new line.
You have two new lines, delete one.

Reply:
/fxd